src/metabase/db/migrations.clj

```diff
@@ -228,13 +234,15 @@
         {admin-group-id :id}     (perm-group/admin)]
     (binding [perm-membership/*allow-changing-all-users-group-members* true]
       (doseq [{user-id :id, superuser? :is_superuser} (db/select [User :id :is_superuser])]
-        (db/insert! PermissionsGroupMembership
-          :user_id  user-id
-          :group_id all-users-group-id)
-        (when superuser?
+        (u/ignore-exceptions
```

Comment:
no this is for things that we want to ignore. We know this will cause an exception if the things are already in the DB so we can safely ignore it